CHANGELOG.md

```diff
@@ -4,6 +4,92 @@
 
 All notable changes to this project will be documented in this file.
 
+## [0.21.0](https://github.com/cds-snc/gcds-components/compare/gcds-components-v0.20.0...gcds-components-v0.21.0) (2024-05-08)
+
+### Breaking changes
+
+#### Transition to form-associated custom elements
+
+We've undertaken significant efforts to migrate our components to form-associated custom elements. GC Design System form components will now use the shadow DOM (document object model). The shadow DOM provides improved encapsulation and control over styling and lets form components integrate with native browser features for validation and accessibility. This feature depends on HTMLElement API attachInternals which is supported in most browsers.
+
+This change will require all form components to have the name attribute. Here's a list of the GC Design System components affected by the change:
+
+- File uploader (`gcds-file-uploader`)
+- Input (`gcds-input`)
+- Select (`gcds-select`)
+- Text area (`gcds-textarea`)
+
+##### Required changes for form components
+
+- Add a `name` attribute
+- Your code should look similar to the following:
+
+``` html
+<gcds-file-uploader name=”” >
+<gcds-input name=”” >
+<gcds-select name=”” >
+<gcds-textarea name=”” >
+```
```

Comment:
Sorry a bit of a nit; but helpful if they copy this section
```suggestion
``` html
<gcds-file-uploader name="" >
<gcds-input name="" >
<gcds-select name="" >
<gcds-textarea name="" >
```
```

Reply:
Actually, now that I'm looking at this again, should we also include `input-id=""` and `label=""` because those are required for each form field?